data/config/game_view.cfg.json

```diff
@@ -16,22 +18,17 @@
             "y": 15
         },
         "dealer": {
-            "x": 600,
-            "y": 50
-        },
-        "player_one_hand": {
-            "x": 600,
+            "x": 550,
             "y": 50
```

Comment:
You have width for player area, do we need one for dealer ?

Reply:
We could, especially if we want to define areas to refresh independently in the game view, as we discussed last time. In that case we will need a height as well. But I'd rather keep this for another time.
For the moment it's pretty much as if the width for dealer area was 0.